pom.xml

```diff
@@ -121,6 +122,7 @@
                         <id>validate</id>
                         <phase>${checkstyle.phase}</phase>
                         <configuration>
```

Comment:
I believe we can remove the configuration block in the execution because it is already stated for the whole plugin

Reply:
Ok, done